supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc

```diff
@@ -1,10 +1,901 @@
 [[red-hat-jboss-eap-conventions]]
 
+// ***********************
+// Terms starting with 'A'
+// ***********************
+
+[discrete]
+[[activemq]]
+==== ActiveMQ (noun)
+*Description*: Do not use "ActiveMQ" by itself to refer to the built-in messaging technology for JBoss EAP.
+
+*Use it*: no
+
+*Incorrect forms*: Active MQ, Active-MQ
+
+*See also*: xref:activemq-artemis[ActiveMQ Artemis], xref:jboss-eap-messaging[JBoss EAP messaging]
+
+[discrete]
+[[activemq-artemis]]
+==== ActiveMQ Artemis (noun)
+*Description*: Use "ActiveMQ Artemis" only when describing the technology used to implement the built-in messaging for JBoss EAP.
+
+*Use it*: with caution
+
+*Incorrect forms*: Active MQ Artemis, Active-MQ Artemis
+
+*See also*: xref:activemq[ActiveMQ], xref:jboss-eap-messaging[JBoss EAP messaging]
+
+// ***********************
+// Terms starting with 'B'
+// ***********************
+
+[discrete]
+[[batch-subsystem]]
+==== Batch Subsystem (noun)
+*Description*: Use "Batch Subsystem" when referring to the xref:batch-jberet[batch-jberet] subsystem in titles and headings. Capitalize each word. See xref:batch-jberet[batch-jberet] entry for the correct usage in general text.
+
+*Use it*: yes
+
+*Incorrect forms*: Batch subsystem, batch subsystem
+
+*See also*: xref:batch-jberet[batch-jberet]
+
+[discrete]
+[[batch-jberet]]
+==== batch-jberet (noun)
+*Description*: The "batch-jberet" subsystem is used to configure and manage batch jobs. Write in lower case as two words separated by a hyphen. See xref:batch-subsystem[Batch Subsystem] entry for the correct usage in headings.
```

Comment:
`lowercase` is one word. Ditto for all later mentions.

Reply:
Fixed